src/common/sign-in/sign-in.tsx

```diff
@@ -80,10 +114,16 @@ export const SignIn = () => {
               || passState === ''
               || emailState.fieldStatusEmail === TextFieldStatus.error
             }
+            onClick={() => {
+              setBtnIsLoading(true);
+              loginUser();
+            }}
           >
-            Ingresar
+            {btnIsLoading ? <SmallSpinnerSVG className={styles.smallSpinner} /> : 'Ingresar'}
```

Comment:
Even if is loading and shows a spinner, it could be clickable. What do you think about disable the button too?